dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -167,6 +168,27 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         }
     }
 
+    void AppNotificationManager::Register()
+    {
+        RegisterHelper(winrt::hstring{} /* displayName */, std::wstring{} /* iconFilePath */);
```

Comment:
Why don't you retrieve the correct displayname and icon path here before calling into the helper? Passing in empty is a strange pattern.

Reply:
Also Register would fail now because you call empty. Please don't use anti patterns and use valid arguments to pass into 
APIs